ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1141,6 +1141,7 @@ ethernet_interfaces:
     dot1x:
       port_control: < "auto" | "force-authorized" | "force-unauthorized" >
       port_control_force_authorized_phone: < true | false >
+      reauthentication: < true | false | default -> false >
```

Comment:
you don't need the default here, since we don't output anything unless the user sets either true or false. So basically we leave it up to EOS default behavior (which we will not document here)

Reply:
resolved